vnctl/lib/vnctl/cli/base.rb

```diff
@@ -94,9 +84,9 @@ def self.define_show
         desc "show [UUID(S)]", "Shows all or a specific set of #{namespace}(s)."
         define_method(:show) do |*uuids|
           if uuids.empty?
-            puts get(suffix)
+            puts Vnctl.webapi.get(suffix)
           else
-            uuids.each { |uuid| puts get("#{suffix}/#{uuid}") }
+            uuids.each { |uuid| puts Vnctl.webapi.get("#{suffix}/#{uuid}") }
```

Comment:
I don't see nested iterators. Perhaps the bot thinks `define_method(:show) do |*uuids|` is also an iteration?